bspump/jupyter/jupyter.py

```diff
@@ -504,7 +506,6 @@ def webserver(app):
 
 if deploy_secret:
     import zipfile
-    from bspump.http.web.server import *
```

Comment:
This conditional import was intentional. Why import the thing if you're not gonna use it? If ruff complains then use a comment like `#noqa` or whatever you need to do to make ruff shut up.